go/test/endtoend/utils/mysql.go

```diff
@@ -96,7 +97,7 @@ func NewMySQLWithMysqld(port int, hostname, dbName string, schemaSQL ...string)
 	}
 	return params, mysqld, func() {
 		ctx := context.Background()
-		_ = mysqld.Teardown(ctx, mycnf, true)
+		_ = mysqld.Teardown(ctx, mycnf, true, 30*time.Second)
```

Comment:
Worth adding a default const for e2e tests too, IMO.

If we hit this value, will the test and workflow fail? If so, might be worth doubling it to avoid failures when CI is just really slow.

Reply:
30 seconds seems long for our tests, but we can increase it if you think that's appropriate? I'll also move it to a constant.